client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -520,6 +523,7 @@ const siteSetupFlow: Flow = {
 
 				case 'goals':
 					// Skip to dashboard must have been pressed
+					setIntent( SiteIntent.Build );
```

Comment:
Sets site_intent to `build' in redux store if user clicks "Skip to dashboard"

Reply:
Im not understanding the need for the other changes, but this seems like the main thing we need here?

Reply:
Yes this is the main change, but I needed to add the other changes in order to get the latest site_intent value.